pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+#Before making the Pull Request:
+
+- [ ] Is there a description to this pull request that points to a specific user story, feature, or bugfix?
+- [ ] Are there comments where the code may be unclear?
+- [ ] Is all dead code, commented out code, etc. removed?
+- [ ] Are there tests written for these changes?
```

Comment:
When do we want to start requiring tests? Don't we need some basic functionality before we can start writing meaningful tests?

Reply:
We will want to start implementing basic tests right away.  The tests themselves can create the "meaningfulness" we want to see in the code (TDD).  However, I think I understand your point as we are still just setting up the basic scaffolding.

Reply:
Do we need to have tests written before we do a PR? what if I'm just doing the basic Layout? no real functionality there yet.

Reply:
I believe we are going to need to revisit some features as we integrate them together, it will be hard to do tests till everything is integrated into the right components

Reply:
Looks like we are missing a check box for commit descriptions

Reply:
I'm ok with requiring tests to merge a pull request once we have the scaffolding set up, but it seems hard to write tests at this point in the project